tendrl/commons/objects/__init__.py

```diff
@@ -94,73 +86,83 @@ def save(self, update=True, ttl=None):
                         _stored_hash = NS._int.client.read(_hash_key).value
                 if self.hash == _stored_hash:
                     # No changes in stored object and current object, dont save current object to central store
-                    return
+                    changed = True
```

Comment:
Instead of breaking all the unit tests for save()

Dont use the `changed` variable and simply call [link] at L89 and at the end of save()

[link]: https://github.com/Tendrl/commons/blob/develop/tendrl/commons/utils/etcd_utils.py#L66